src/components/App.tsx

```diff
@@ -737,17 +748,15 @@ export class App extends React.Component<any, AppState> {
         decryptedData: SocketUpdateDataSource["SCENE_INIT" | "SCENE_UPDATE"],
       ) => {
         const { elements: remoteElements } = decryptedData.payload;
-        const restoredState = restore(remoteElements || [], null, {
-          scrollToContent: true,
-        });
```

Comment:
Scrapped `restore` which I wanted to get rid of here anyway, but for this PR it's a necessity because we need to sync even invisible elements. Case: when user syncs text deletion (empty text).

We shouldn't need to use this API → elements that are being synced should be fully normalized.